apps/docs/src/app/platform/component-docs/platform-link/platform-link-examples/platform-link-standard-example.component.html

```diff
@@ -1,64 +1,57 @@
 <p>
-    This is an Example of
-    <fdp-link [href]="'http://www.google.com'" [title]="'Extra info as tooltip text'">
-        <fd-icon [glyph]="'chain-link'"></fd-icon>
-        Standard Link
+    This is an example of a
+    <fdp-link href="http://www.google.com" [title]="'Extra info as tooltip text'">
+        <fd-icon glyph="chain-link"></fd-icon>
+        standard
     </fdp-link>
-    text after link
+    link.
 </p>
 <br />
 
 <div>
-    This is an Example of
-    <fdp-link href="http://www.google.com" [title]="'Extra info as tooltip text'" linkType="subtle">
-        Subtle Link
-    </fdp-link>
-    text after link
+    This is an example of a
+    <fdp-link href="http://www.google.com" [title]="'Extra info as tooltip text'" linkType="subtle"> subtle </fdp-link>
+    link.
 </div>
 <br />
 
 <p>Example of a text link.</p>
-<fdp-link [id]="'regular-text-link'" [href]="'fundamental-ngx#/platform/home'" [title]="'Regular link description'">
-    text link
+<fdp-link [id]="'regular-text-link'" href="fundamental-ngx#/platform/home" [title]="'Regular link description'">
+    text
 </fdp-link>
 <br />
 <br />
 <p>Example of an external link.</p>
-<fdp-link [href]="'https://www.google.com/'" [title]="'opens a external webpage'" [target]="'_blank'">
+<fdp-link href="https://www.google.com" [title]="'opens a external webpage'" target="_blank">
     navigates to external webpage
 </fdp-link>
 <br />
 <br />
 <p>Example of an internal page link.</p>
-<fdp-link [href]="'fundamental-ngx#/platform/home'" [title]="'opens a internal page'" [target]="'_blank'">
+<fdp-link href="fundamental-ngx#/platform/home" [title]="'opens a internal page'" target="_blank">
     navigates to internal page
 </fdp-link>
 <br />
 <br />
 <p>Example of opening link in new tab.</p>
-<fdp-link [id]="'new-tab-link'" [href]="'https://www.google.com/'" [title]="'opens in new tab'" [target]="'_blank'">
+<fdp-link [id]="'new-tab-link'" href="https://www.google.com" [title]="'opens in new tab'" target="_blank">
```

Comment:
Just curious, why is the binding for `title` not working?

Reply:
Hi @InnaAtanasova 
Please look at my answer above 🙂
https://github.com/SAP/fundamental-ngx/pull/7201#discussion_r754208558